prep-html-script-tag/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE >
+<html>
+  <head>
+    <title>HTML Script Tag</title>
+  </head>
+  <body>
+    <script>
+      console.log('I have added JS to a script tag in HTML!');
+    </script>
+    <script src="main.js"></script>
+    <script>
+      console.log('I have added a script file via the src attribute!');
```

Comment:
This console log should be in the main.js file without the script tag

the script tag above with the src attribute will load the JavaScript from the main.js file.